src/tests/regressionTests.test.tsx

```diff
@@ -1088,15 +1089,15 @@ describe("regression tests", () => {
     assertSelectedElements(rect3);
   });
 
-  it("should show fill icons when element has non transparent background", () => {
+  it("should show fill icons when element has non transparent background", async () => {
     UI.clickTool("rectangle");
     expect(screen.queryByText(/fill/i)).not.toBeNull();
     mouse.down();
     mouse.up(10, 10);
     expect(screen.queryByText(/fill/i)).toBeNull();
 
     UI.clickLabeledElement("Background");
-    UI.clickLabeledElement(t("colors.fa5252"));
+    UI.clickOnTestId("color-red");
```

Comment:
can't find it because it seems that the `UI.clickLabeledElement("Background");` on previous line no longer opens the color picker, but rather closes the whole properties sidebar because it's no longer queryable, as if it deselected the element (though it doesn't).

Reply:
The tests are fixed now, this was due to `ResizeObserver` not getting mocked which was required by radix popover